tests/pdf_consts.py

```diff
@@ -23,3 +23,5 @@ def file(filename):
 rgb_image_pdf = file('tests/test_pdfs/rgb_image.pdf')
 
 cmyk_and_rgb_images_in_one_pdf = file('tests/test_pdfs/cmyk_and_rgb_in_one_pdf.pdf')
+
+new_address_margin = file('tests/test_pdfs/new_address_margin.pdf')
```

Comment:
It doesn't really matter, but I'd just call it `address_margin`. It's only new right now, but is unlikely to be renamed in the future.